docs/user_guide/Certificates/Official_Certificates.inc

```diff
@@ -1,38 +1,54 @@
 Installing Official Certificates
 --------------------------------
 
-Below are the steps to install official certificates for a SIMP client on
-the SIMP server:
+This section describes how to install infrastructure certificates from an
+official certificate authority on the puppetserver for distribution to client
+servers.  You need to have simp_options::pki set to ``simp`` on the client for
+this to work.
+
+The key distribution directory on the puppetserver is located under the site_files directory.
```

Comment:
site_files instances need to be double back quoted just like other references to directories.